src/OpenLoco/src/Vehicles/Vehicle2.cpp

```diff
@@ -40,18 +40,17 @@ namespace OpenLoco::Vehicles
     static bool shouldStartWheelSlipping(const Vehicle& train, VehicleBogie& frontBogie)
     {
         const auto* vehObject = ObjectManager::get<VehicleObject>(frontBogie.objectId);
-        if (train.head->status != Status::travelling)
+        if (!vehObject->hasFlags(VehicleObjectFlags::canWheelslip) || vehObject->power == 0)
```

Comment:
any reason why you changed the order here? I don't mind, just wondering.

Reply:
Nevermind about the Var_5A, that should be a different PR.